src/config/config.ts

```diff
@@ -1,7 +1,4 @@
 import convict from "convict"
-import * as dotenv from "dotenv"
-
-dotenv.config()
```

Comment:
Just to check in, was this deletion of code intended?

Reply:
yea, convict does the env load automatically, we dont need dotenv